### PR TITLE
docs: Update README file

### LINKED ISSUE #23: doc: update ReadMe file
## Description
The versions of Django and Django rest framework used for the project Open source programs(OSP) is not mentioned in the readme file and need to be added.

versions used are: 
Django 3.0.7
Django rest framework 3.11.0  
 
For additional information refer requirements.txt file.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
15 mins

### PR DESCRIPTION
**Pull request by vj-codes**
### Description

Updated README.md file with the versions of Django and Django rest framework used for the project Open source programs(OSP) 

Fixes #23 

### Type of Change:
- Documentation

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 Open Source Programs (OSP) is an application that simplifies the processing and selection procedure of Open Source Programs of AnitaB.org Open Source or other third-party programs. This is the Backend repo for OSP.
 
 ## Tech Stack
-- Django
-- Django REST Framework
+- Django 3.0.7
```
Comment (Monal5031) [inline README.md:5]: IMO we should mention the python version as well

**Review by Monal5031 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bismitaguha) [reply README.md:5]: Yes same. @vj-codes Please make the change.

Comment (vj-codes) [reply README.md:5]: python 3.8.3  after rest frame work right? @bismitaguha @Monal5031

Comment (bismitaguha) [reply README.md:5]: 3.7 or 3.7+ should work. @Monal5031 your opinion.

Comment (Monal5031) [reply README.md:5]: Agreed. 3.7+ should be good

**Review by bismitaguha — APPROVED** [review]

**Pull request merged**